pkg/process/util/address.go

```diff
@@ -24,11 +25,24 @@ func AddressFromNetIP(ip net.IP) Address {
 	return a
 }
 
-// AddressFromString creates an Address using the string representation of an v4 IP
+// AddressFromString creates an Address using the string representation of an IP
 func AddressFromString(ip string) Address {
 	return AddressFromNetIP(net.ParseIP(ip))
 }
 
+// AddressFromByteString creates an Address using the byte string representation of an IP
+func AddressFromByteString(ip string) Address {
+	if len(ip) == 16 {
+		return V6AddressFromBytes([]byte(ip))
+	}
+	return V4AddressFromBytes([]byte(ip))
+}
+
+// NetIPFromIPByteString returns a net.IP from an Address
```

Comment:
nit: doc should say "from a byte string".   

Also: this seems to only be used for tests -- why not use `AddressfromByString().IP()`